epersonregistrations.md

```diff
@@ -44,13 +44,16 @@ To create a new EPerson registration, perform a post with the JSON below to the
 }
 ```
 
+Requires query parameter 'type' set with either 'register' or 'forgot' value, depending on the action requested.
+
 No other properties can be set (e.g. the name cannot be defined)
 If successful, an email will be sent with a token allowing the user to continue the registration
 
 Verifying whether a new registration can be created can happen using the "epersonRegistration" [feature](features.md), verified against the site
 
 Status codes:
 * 201 Created - if the operation succeed
+* 400 Bad Request - if e.g. the e-mail address is in a domain that is not allowed in config `authentication-password.domain.valid`
 * 401 Unauthorized - if registration is disabled, you are not authorized to create a new registration
 * 422 Unprocessable Entity - if the email address was omitted
```

Comment:
violation of complex business rule, like an email address outside the allowed ones should fail with a 422. If angular implement a client side validation for not empty email this would mean just invalid domain for our preferred "rest client"

Reply:
Missing accountRequestType query param or value in it that is not forgot nor register will result in 400 bad request error